pkg/input/survey/survey_test.go

```diff
@@ -0,0 +1,20 @@
+package survey_test
```

Comment:
Tests in WIP.

Reply:
Can reproduce the error with the old code with tests, when default is an empty string.